.gitignore

```diff
@@ -12,6 +12,10 @@ cypress/results/*
 cypress/reports/*
 cypress/screenshots/*
 cypress/videos/*
+...
+cypress/integration/examples/*
+...
```

Comment:
these points were in the guide as a reference that there is more content below and above the line that needs to be added. Please remove them :)

Reply:
Points deleted.